raiden/network/rpc/client.py

```diff
@@ -709,6 +709,10 @@ def manager(self, manager_address):
         return self.address_manager[manager_address]
 
     def manager_by_asset(self, asset_address):
+        if asset_address not in self.asset_manager:
+            self.asset_manager[asset_address] = manager
```

Comment:
Good catch... I probably added that in the very beginning... will try removing it because it might not be needed at all!